connectors/google-sheets/element-templates/google-sheets-connector.json

```diff
@@ -0,0 +1,586 @@
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.index"
+      },
+      "constraints": {
+        "notEmpty": false,
+        "maxLength": 100,
+        "pattern": {
+          "value": "^([0-9]*$)|(secrets.*$)",
+          "message": "Must contains only numbers."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "createRow"
+      }
+    },
+    {
+      "label": "Column index",
+      "description": "Enter the index of the column",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.index"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "maxLength": 100,
+        "pattern": {
+          "value": "[A-Z]+",
+          "message": "Must contains only capital letters."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "deleteColumn"
+      }
+    },
+    {
+      "label": "Start index",
+      "description": "Enter start index (leave empty if add to the end of the sheet)",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.startIndex"
+      },
+      "constraints": {
+        "notEmpty": false,
+        "maxLength": 100,
+        "pattern": {
+          "value": "^([0-9]*$)|(secrets.*$)",
+          "message": "Must be a number."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "addEmptyColumnOrRow"
+      }
+    },
+    {
+      "label": "End index",
+      "description": "Enter End index (leave empty if add to the end of the sheet)",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.endIndex"
+      },
+      "constraints": {
+        "notEmpty": false,
+        "maxLength": 100,
+        "pattern": {
+          "value": "^([0-9]*$)|(secrets.*$)",
+          "message": "Must be a number."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "addEmptyColumnOrRow"
+      }
+    },
+    {
+      "label": "Cell ID",
+      "description": "Enter the ID of the cell ",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.cellId"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "maxLength": 100
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "addValues"
+      }
+    },
+    {
+      "label": "Enter values",
+      "description": "Enter values for the row",
```

Comment:
can you add here an example, that the user don't need to lookup the format

Reply:
<img width="378" alt="image" src="https://github.com/camunda/connectors-bundle/assets/131355515/679a210f-9cb7-45f1-810e-f6ab6fd9afbf">